user/views.py

```diff
@@ -4,14 +4,15 @@
 from django.contrib import messages
 from .models import User
 from django.http import Http404
-
+from tweet.models import Tweet
 
 
 
 
 def home(request):
   if request.user.is_authenticated:
-    return render(request,'home.html')
+    all_tweets = Tweet.objects.all().order_by('-id')
+    return render(request,'home.html', {'all_tweets': all_tweets})
```

Comment:
So, if a user has 500000 tweets you will load all on the page ?

Reply:
yes Its done for starting but for future I think to load with ajax scroll as I've done this on abother project. I did like this because you say to work simple first.